org.scram-pra.scram.yml

```diff
@@ -0,0 +1,23 @@
+app-id: org.scram-pra.scram
+runtime: org.kde.Sdk
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: scram-gui
+rename-icon: scram
+rename-desktop-file: scram.desktop
+finish-args:
+- "--socket=wayland"
+- "--socket=fallback-x11"
+- "--filesystem=home"
+modules:
+- name: scram
+  buildsystem: cmake-ninja
+  build-commands:
+  - desktop-file-install --dir=/app/share/applications cutecom.desktop
+  - install -Dm755 org.scram-pra.scram.appdata.xml /app/share/metainfo/org.scram-pra.scram.appdata.xml
+  - install -Dm755 gui/images/scram_log.svg /app/share/icons/hicolor/scalable/apps/scram.svg
+  sources:
+  - type: git
+    url: https://github.com/Martin2278/scram
```

Comment:
this appear to be a fork. Why not using upstream directly?

Reply:
the creator of the project does not react on any messages. I changed also some things and I will do it also in the future